assets/src/edit-story/app/font/fontProvider.js

```diff
@@ -105,18 +117,39 @@ function FontProvider({ children }) {
     [getFontByName]
   );
 
+  const getMenuFonts = useCallback((fontList) => {
```

Comment:
I really think we should not be requesting fonts we have loaded before. Should be have a small piece of state that is an array of `loadedFonts` only make request if there is a diff from loadedFonts and requested fonts? I don't like making requests that not required. It is a waste of resources.

Reply:
Added new logic to checked the loaded font faces for font family. Now when scrolling new font faces will be loaded through api. 
And I'm not sure what the question 2) means. We are getting font faces as string for some list of fonts (Visible options). And then inject that font faces.